### PR TITLE
remove job from pull and import

### PR DESCRIPTION
**Pull request by LK4D4**
Closes #12396

### CODE REVIEW

**Comment by LK4D4** [comment]
ping @calavera @duglin @tiborvass

**Comment by calavera** [comment]
LGTM

File: api/server/server.go [hunk 1]
```diff
@@ -661,31 +663,45 @@ func postImagesCreate(eng *engine.Engine, version version.Version, w http.Respon
...
-		streamJSON(job, w, true)
-	} else {
-		job.Stdout.Add(utils.NewWriteFlusher(w))
-	}
-	if err := job.Run(); err != nil {
-		if !job.Stdout.Used() {
+		src := r.Form.Get("fromSrc")
+		imageImportConfig := &graph.ImageImportConfig{
+			Changes:   r.Form["changes"],
+			InConfig:  r.Body,
```
Comment (calavera) [inline api/server/server.go]: I think that `InConfig` can be decoded with `runconfig.DecodeContainerConfig`

**Comment by duglin** [comment]
LGTM
@tiborvass for the merge

File: api/server/server.go (hunk 1 above)
Comment (LK4D4) [inline api/server/server.go]: InConfig actually not `config`. It is `archive`

**Comment by calavera** [comment]
It looks like this has a small conflict with #12457. I'll rebase that pr when this is in master.

**Comment by cpuguy83** [comment]
Even TestPullVerified worked, LGTM :)

**Pull request merged**